eng/Build.props

```
@@ -4,7 +4,7 @@
     <TargetingPackageProject Include="$(RepoRoot)src\targetPacks\ILsrc\**\*.csproj" />
     <ProjectToBuild Include="@(TargetingPackageProject)" />
 
-    <TextOnlyPackageProject Include="$(RepoRoot)src\textOnlyPackages\src\**\*.csproj" />
+    <TextOnlyPackageProject Include="$(RepoRoot)src\textOnlyPackages\src\**\*.csproj" Exclude="$(RepoRoot)src\textOnlyPackages\src\**\Content\**\*.csproj" />
```

I think clamping down the `Include` is little more maintainable/readable than adding wildcard `Exclude`s:

```suggestion
    <TextOnlyPackageProject Include="$(RepoRoot)src\textOnlyPackages\src\*\*\*.csproj" />
```

---

Yeah, that does seem better, thanks.